recipe/bld.bat

```diff
@@ -9,6 +9,7 @@ pushd "%SRC_DIR%"\build
 cmake -G Ninja ^
       -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
       -DCMAKE_BUILD_TYPE=Release ^
+      -DTILEDB_CMAKE_IDE=ON ^
```

Comment:
Why this option instead of `TILEDB_SUPERBUILD=OFF`? If there's some reason for it, please add comments.

Reply:
`-DTILEDB_SUPERBUILD=OFF` disables auto-downloading vcpkg. Added comments.

Reply:
@ihnorton does that address your concern?